### PR TITLE
various fixes to analysis

### PR DESCRIPTION
**Pull request by kepper**
This is a small revision of things in the analysis module. I'm not claiming that everything is fine now, but I hope it's better than before ;-) This should go into v4 pretty soon, even though more work need to be done afterwards. @musicEnfanthen, could you please have a quick look? Thanks very much…

### CODE REVIEW

File: _guidelines-v4/10-analysisharm/01-analysis/01-analysisdescribingrelationships.md [hunk 1]
```diff
@@ -6,28 +6,30 @@ version: "v4"
...
 
 {% include desc atts="att.linking/copyof
+att.linking/sameas
 att.linking/corresp
 att.linking/next
 att.linking/prev
-att.linking/sameas
+att.linking/precedes
+att.linking/follows
 att.linking/synch" %}
```
Comment (musicEnfanthen) [inline _guidelines-v4/10-analysisharm/01-analysis/01-analysisdescribingrelationships.md:18]: Is this order related to anything? Wouldn't it be better to have it in alphabetical order?

File: _guidelines-v4/10-analysisharm/01-analysis.md [hunk 2]
```diff
@@ -6,4 +6,6 @@ title: "Analytical Information"
...
+This chapter describes the use of attributes that capture data which may be useful for analytical purposes. The analysis module provides attributes that record relationships between entities found in the encoding. These attributes may be used differently by different users, depending on the purpose of the analysis.
+
+These Guidelines recommend that encoders employ commonly accepted analytical practices, such as ‘functional analysis’ or ‘Schenkerian analysis’, and document their use in the {% include link elem="encodingDesc" %} described in section {% include link id="headerEncodingDescription" %}. For general information on musical analysis, please consult Grove Music Online, ‘[Analysis](https://doi.org/10.1093/gmo/9781561592630.article.41862){:.link_ref}’.
```
Comment (musicEnfanthen) [inline _guidelines-v4/10-analysisharm/01-analysis.md:11]: Memo for later: We should find an external link to musical analysis that is not behind a subscription wall.

File: _includes/v4/examples/analysis/analysis-sample173.txt [hunk 3]
```diff
@@ -25,7 +20,7 @@
       <tuplet copyof="#analysis.tup1" xml:id="analysis.tup4"/>
     </layer>
   </staff>
-  <staff n="3">
+  <staff n="2">
```
Comment (musicEnfanthen) [inline _includes/v4/examples/analysis/analysis-sample173.txt:23]: Since you removed the first mrest layer, would it make sense to remove the last one, too?

File: _guidelines-v4/10-analysisharm/01-analysis/01-analysisdescribingrelationships.md (hunk 1 above)
Comment (kepper) [reply _guidelines-v4/10-analysisharm/01-analysis/01-analysisdescribingrelationships.md:18]: This was requested by someone (apologies for not knowing who it was right now…). I think this order makes sense, in that it order the attributes by relevance / frequency of usage. Yes, we can argue if `@copyof` should really go first or `@sameas`, but I think it's clear that together, they should precede `@next` and `@prev`, etc. So this is intentional, and I hope it's more instructive as well…

File: _includes/v4/examples/analysis/analysis-sample173.txt (hunk 3 above)
Comment (kepper) [reply _includes/v4/examples/analysis/analysis-sample173.txt]: yes, makes perfect sense. The idea was to simplify the example to show only what's relevant here. Resolved in a3789930814065eb98d396cfc0278cf7bad8d9b2

File: _guidelines-v4/10-analysisharm/01-analysis.md (hunk 2 above)
Comment (kepper) [reply _guidelines-v4/10-analysisharm/01-analysis.md:11]: may we turn that into a separate issue and assign it to you? ;-)

File: _guidelines-v4/10-analysisharm/01-analysis/01-analysisdescribingrelationships.md (hunk 1 above)
Comment (musicEnfanthen) [reply _guidelines-v4/10-analysisharm/01-analysis/01-analysisdescribingrelationships.md:18]: I am fine if it makes sense for you ;)

File: _guidelines-v4/10-analysisharm/01-analysis.md (hunk 2 above)
Comment (musicEnfanthen) [reply _guidelines-v4/10-analysisharm/01-analysis.md:11]: Yes, please, but I do not insist on the assignment.

**Comment by musicEnfanthen** [comment]
Thanks a lot for your work!

**Comment by kepper** [comment]
thanks for merging :-)

**Comment by kepper** [comment]
this fixes #89
